build-package

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+cd $(dirname ${BASH_SOURCE[0]})
+
+# Some variables
+DEBVERSION=1.99.5-1
```

Comment:
Because I'm not sure if we do 2.0.0-pre6 it will grab 2.0.0 as an update. It seems like previous.99 is the way they do it, but I might be wrong. The `-1` is a distro-specific extra number. So if you gave 1.5.0-4, it means it's the 4th rebuilt of the package. This is standard and every package is that way.